session01/break_me.py

```diff
@@ -0,0 +1,46 @@
+#!/Library/Frameworks/Python.framework/Versions/3.6/bin/python
+# Program tests NameError, TypeError, SyntaxError, AttributeError
+import os
+# NameError test:
+def name_test():
+	''' This tests NameError '''
+	try: 
+		print(brian)
+	except NameError as e:
+		print("Your variable name has not been defined: ", e)
+
+# TypeError test:
+def type_test(val):
+	''' This tests TypeError '''
+	try:
+		a=val/4
+	except TypeError as e:
+		print("Your type is incorrect:", e)
+
+# SyntaxError test:
+def syntax_test():
+	''' This tests SyntaxError '''
+	try:
+		date = eval('datetime(2009, 12a, 31)')
+	except SyntaxError as e:
+		print("Your syntax is wrong: ", e)
+
```

Comment:
I like how you got around that syntax errors are usually found on loading, rather than during runtime!

Reply:
I just completed the https://uwpce-pythoncert.github.io/IntroPython-2017/modules/GitWorkflow.html#git-workflow process to make this change.  Pls let me know if I succeeded.

Reply:
that was clever -- but do remember that using "eval" in production code is discouraged, mostly for security reasons.

works well for this though.